oauth/readme.md

```diff
@@ -48,14 +48,15 @@ Your `CLIENT_ID` and `CLIENT_SECRET` can be retrieved from the Adobe Developer C
 
 Adobe OAuth 2.0 requires SSL, so you will need to create a self-signed cert using the OpenSSL CLI.
 
-Be sure to run the OpenSSL CLI command in the repo's `bin` directory:
+If you don't have the `mkcert` tool installed already, install it via Homebrew.
```

Comment:
Homebrew won't work on Windows. Let's maybe rewrite this to be a little more "generic":

```suggestion
If you don't have the `mkcert` tool installed already, [install it](https://github.com/FiloSottile/mkcert#installation) on your system and run `mkcert -install`.
```